stdlib/public/Platform/Platform.swift

```diff
@@ -187,6 +209,7 @@ public func open(
   return _swift_Platform_open(path, oflag, 0)
 }
 
+#if !os(Windows) || CYGWIN
 public func open(
   _ path: UnsafePointer<CChar>,
   _ oflag: Int32,
```

Comment:
Sorry, but I think it calls `_swift_Platform_open`: https://github.com/compnerd/apple-swift/blob/34777d0718cce6604d52353c25b69b48f32392e7/stdlib/public/Platform/Platform.swift#L218